src/views/CreateFacility.vue

```diff
@@ -188,9 +188,9 @@ export default defineComponent({
         } else {
           throw resp.data;
         }
-      } catch (error) {
+      } catch (error: any) {
         logger.error(error)
-        showToast(translate('Failed to create facility.'))
+        showToast(error.response.data.error?.message)
```

Comment:
Handle the case that when the error is not found display the default message of `Failed to create facility.`